library/model/book.py

```diff
@@ -35,24 +23,9 @@ def __init__(self, title, authors, publisher, pub_date, genres, pages, isbn, typ
         self.duration = duration
         self.existing_items = existing_items
         self.borrowed_items = borrowed_items
+        # self.isBorrowable = self.can_borrow()
 
     @classmethod
-    def from_borrowed_book(cls, borrowed_book: "BorrowedBook") -> "Book":
-        book = Book(
-            borrowed_book.title,
-            borrowed_book.authors,
-            borrowed_book.publisher,
-            borrowed_book.publication_date,
-            borrowed_book.genres,
-            borrowed_book.pages,
-            borrowed_book.isbn,
-            borrowed_book._book_type,
-            borrowed_book.duration,
-            borrowed_book.existing_items,
-            borrowed_book.borrowed_items,
-        )
-        return book
-
     def can_borrow(self) -> bool:
         if self._book_type == "Paper":
```

Comment:
You could consider creating new classes for the different types of books, which all inherit from an abstract book class.